### ISSUE TITLE
implement create?

### ISSUE THREAD

**Issue by hananbeer** [oguimbal/HyVM#2]
is there any particular reason create is not implemented?

**Comment by oguimbal** [comment]
no, not really, that's right (at least no other than "there is no way that our user will need that").
I'll add it.
